origin/proxy.go

```diff
@@ -239,7 +239,7 @@ func (p *proxy) logRequest(r *http.Request, fields logFields) {
 		p.log.Debug().Msgf("All requests should have a CF-RAY header. Please open a support ticket with Cloudflare. %s %s %s ", r.Method, r.URL, r.Proto)
 	}
 	p.log.Debug().Msgf("CF-RAY: %s Request Headers %+v", fields.cfRay, r.Header)
-	p.log.Debug().Msgf("CF-RAY: %s Serving with ingress rule %v", fields.cfRay, fields.rule)
+	p.log.Debug().Msgf("CF-RAY: %s Serving (%s, %s) with ingress rule %v", r.Host, r.URL.Path, fields.cfRay, fields.rule)
```

Comment:
This is a good idea, but why not just add those fields as structured log values (and move the ingress rule %v into a structured log field as well)?

Something like:
```go
    p.log.Debug().
        Str("CF-RAY", fields.cfRay).
        Str("host", r.Host).
        Str("path", r.URL.Path).
        Str("rule", fields.rule).
        Msg("Inbound request")
```

This would be more idiomatic for what we would like to achieve.